VSCode/armsnippets.json

```diff
@@ -506,6 +506,123 @@
     ],
     "description": "Availability Set"
 },
+"Azure Firewall": {
+    "prefix": "arm-firewall",
+    "body": [
+        "{",
+        "    \"name\": \"${1:myAzureFirewall}\",",
+        "    \"type\": \"Microsoft.Network/azureFirewalls\",",
+        "    \"apiVersion\": \"2019-04-01\",",
+        "    \"location\": \"[resourceGroup().location]\",",
+        "    \"tags\": {},",
+        "    \"properties\": {",
+        "       \"applicationRuleCollections\": [",
+        "           {",
+        "               \"name\": \"${1:string}\",",
```

Comment:
Please provide good default values for all tokens (e.g. "applicationRuleCollection1" rather than "string")